dosed/datasets/dataset.py

```diff
@@ -140,10 +141,21 @@ def __init__(self,
                         "label": label,
                     }
 
+                    for start, duration in zip(*data):
+                        valid_index = range(int(max(0, start - self.window_size + 1)),
```

Comment:
I do not agree with these index for me it should start from (disregarding edges):

stop - duration * self.minimum_overlap - self.window_size

to 

start +  duration * self.minimum_overlap + 1